add-ons/istio/README.adoc

```diff
@@ -0,0 +1,62 @@
+:linkattrs:
+
+= Istio Add-on
+
+An addon that will deploy https://Istio.io/[Istio] service mesh on to OpenShift provided by Minishift.
+
+Verify that you have installed the add-on by following the https://github.com/minishift/minishift-addons[Minishift add-ons]
```

Comment:
This sentence is incomplete.

Reply:
Yes, it should be 

Verify that you have installed the add-on by following link:../../README.adoc#download-use-community-addons[general readme].

Reply:
well i would remove the line 7 , as that does not have any value